agent/Gopkg.toml

```diff
@@ -27,7 +27,7 @@
 
 [[constraint]]
   name = "github.com/cihub/seelog"
-  revision ="0fa7e41d2b9600ca6b7e1592478124e4f800f724"
+  version ="2.6"
```

Comment:
was this a manual edit? Curious to know the best practices in dep land.

Reply:
yea this was a manual edit, we always want to constrain to fixed release version where possible. The generated "Gopkg.lock' file will map it to the appropriate [hash here](https://github.com/aws/amazon-ecs-agent/pull/1232/files#diff-0f11b02fdde503182f627b67a8f1876bR66).

Reply:
thanks for that explanation. probably a good time to update my `dep` from `dev` version to something else as well.

Reply:
updated PR summary with dep version info, we should include that information in PRs for dep generated changes to the repo.